ui/selectors/selectors.js

```diff
@@ -3034,6 +3035,17 @@ export const getUpdatedAndSortedAccounts = createDeepEqualSelector(
   },
 );
 
+export const getUpdatedAndSortedAccountsWithCaipAccountId =
+  createDeepEqualSelector(getUpdatedAndSortedAccounts, (accounts) => {
+    return accounts.map((account) => {
+      const { namespace, reference } = parseCaipChainId(account.scopes[0]);
```

Comment:
it seems like scopes will always be defined based on the keyring-interal-api struct but I'm wonder if we should have some guards around this just in case?

Reply:
I didn't run into an issue with scopes being undefined during testing/development, so I vote no